RadarSDK/Radar.h

```diff
@@ -121,56 +122,63 @@ typedef NS_ENUM(NSInteger, RadarRouteUnits) {
  
  @see https://radar.io/documentation/sdk#ios-foreground
  */
-typedef void(^ _Nullable RadarLocationCompletionHandler)(RadarStatus status, CLLocation * _Nullable location, BOOL stopped);
+typedef void (^_Nullable RadarLocationCompletionHandler)(RadarStatus status, CLLocation *_Nullable location, BOOL stopped);
 
```

Comment:
I use a clang format CLI locally. 

However, it would format the entire file, not just the code blocks I added.